concurrency/pkg/apis/concurrency/v1alpha1/concurrency_types.go

```diff
@@ -47,6 +47,13 @@ type ConcurrencySpec struct {
 	Strategy string `json:"strategy,omitempty"`
 	// + optional
 	Selector metav1.LabelSelector `json:"selector,omitempty"`
+	// Label keys that identify the concurrency group of a PipelineRun.
```

Comment:
interesting. I think this is promising - why are we grouping only by keys vs a key-value pair?

Reply:
The example use case here is "only one running pipelinerun per pull request at a time". Here, the pipelinerun (tep linter) is labeled with "tekton.dev/pr-number: <pr number>". This allows us to create 1 concurrency control in total instead of 1 per pull request.

Reply:
https://tektoncd.slack.com/archives/C01D8GL0FM3/p1666885892204219